lib/providers/services/http_service.dart

```diff
@@ -838,6 +845,31 @@ class HttpService {
       return VoteResult(0, false, error: true);
     }
   }
+
+
+  Future<List<String>> getReferredAccounts({String username, limit = 100}) async {
+    username = username ?? userAccount;
+    var result = await getTableRows(
+      code: "accts.seeds",
+      scope: "accts.seeds",
+      table: "refs",
+      value: username,
+      keyType: "name",
+      indexPosition: 2,
+      limit: limit
+    );
+    return List<String>.of(result.map((e) => e["invited"]));
+  }
+
+  Future<bool> isResidentOrCitizen(String username) async {
```

Comment:
is resident or citizen is not a good name for a func that returns a boolean. 

Waiter: Would you like pizza or pasta?  
Me: YES